/**:
  ros__parameters:
    # Plugin activation flags
    use_akima_spline_interpolation: true
    use_eb_smoother: false
    use_qp_smoother: false
    fix_invalid_points: true
    extend_trajectory_backward: true
